src/core/model_config.proto

```diff
@@ -909,4 +935,13 @@ message ModelConfig
   //@@     are made available to custom backends.
   //@@
   map<string, ModelParameter> parameters = 14;
+
+  //@@  .. cpp:var:: ModelWarmUp model_warm_up
+  //@@
+  //@@     Warm up setting of this model. If specified, all instances
+  //@@     will be run with a request sample before serving the model.
+  //@@     This field can only be specified if the model is not an ensemble
+  //@@     model.
+  //@@
+  ModelWarmUp model_warm_up = 16;
```

Comment:
I think this should be "repeated" so that users can specify multiple warmup trials... for example, with different input dimensions.